ui/appui-react/src/appui-react/statusfields/MessageCenter.tsx

```diff
@@ -5,227 +5,165 @@
-import { MessageCenter } from "../layout/footer/message-center/Indicator";
+import { OutputMessagePriority } from "@itwin/core-frontend";
 import { MessageCenterMessage } from "../layout/footer/message-center/Message";
-import { MessageCenterTab } from "../layout/footer/message-center/Tab";
 import { MessageManager } from "../messages/MessageManager";
+import { TitleBar } from "../layout/footer/dialog/TitleBar";
 import type { NotifyMessageDetailsType } from "../messages/ReactNotifyMessageDetails";
-import { StatusBar } from "../statusbar/StatusBar";
-
-/** Enum for the [[MessageCenterField]] active tab
- * @internal
- */
-enum MessageCenterActiveTab {
-  AllMessages,
-  Problems,
-}
-
-/** State for the [[MessageCenterField]] React component
- * @internal
- */
-interface MessageCenterState {
-  activeTab: MessageCenterActiveTab;
-  target: HTMLDivElement | null;
-  messageCount: number;
-  isOpen: boolean;
-}
+import "../layout/footer/message-center/Dialog.scss";
+import "../layout/footer/Indicator.scss";
 
 /** Message Center Field React component.
  * @public
  */
-export class MessageCenterField extends React.Component<
-  CommonProps,
-  MessageCenterState
-> {
-  private _indicator = React.createRef<HTMLDivElement>();
-  private _title = UiFramework.translate("messageCenter.messages");
-  private _unloadMessagesUpdatedHandler?: () => void;
-  private _removeOpenMessagesCenterHandler?: () => void;
-
-  constructor(p: CommonProps) {
-    super(p);
-
-    this.state = {
-      activeTab: MessageCenterActiveTab.AllMessages,
-      target: null,
-      messageCount: MessageManager.messages.length,
-      isOpen: false,
-    };
-  }
-
-  /** @internal */
-  public override componentDidMount() {
-    this._unloadMessagesUpdatedHandler =
-      MessageManager.onMessagesUpdatedEvent.addListener(
-        this._handleMessagesUpdatedEvent,
-        this
-      );
-    this._removeOpenMessagesCenterHandler =
-      MessageManager.onOpenMessageCenterEvent.addListener(
-        this._handleOpenMessageCenterEvent,
-        this
-      );
-    MessageManager.registerAnimateOutToElement(this._indicator.current);
-  }
-
-  /** @internal */
-  public override componentWillUnmount() {
-    // istanbul ignore else
-    if (this._unloadMessagesUpdatedHandler) {
-      this._unloadMessagesUpdatedHandler();
-      this._unloadMessagesUpdatedHandler = undefined;
-    }
-    // istanbul ignore else
-    if (this._removeOpenMessagesCenterHandler) {
-      this._removeOpenMessagesCenterHandler();
-      this._removeOpenMessagesCenterHandler = undefined;
-    }
-  }
-
-  private _handleMessagesUpdatedEvent = () => {
-    // istanbul ignore else
-    if (this._unloadMessagesUpdatedHandler)
-      this.setState({ messageCount: MessageManager.messages.length });
+export const MessageCenterField: React.FC = () => {
+  const [messages, setMessages] = React.useState(MessageManager.messages);
+  const [notify, setNotify] = React.useState("");
```

Comment:
Seems like this can be removed? [Use computed](https://react.dev/learn/choosing-the-state-structure#avoid-redundant-state)